.buildkite/dagster-buildkite/dagster_buildkite/steps/integration.py

```diff
@@ -219,7 +235,6 @@ def build_integration_suite_steps(
         Union[List[AvailablePythonVersion], UnsupportedVersionsFunction]
     ] = None,
 ) -> List[BuildkiteTopLevelStep]:
-    pytest_extra_cmds = pytest_extra_cmds or default_integration_suite_pytest_extra_cmds
```

Comment:
looks like we were installing celery on several of our integration test suites before, even if they did not use celery